src/apiCalls.js

```diff
@@ -0,0 +1,23 @@
+const getAllMovies = () => {
```

Comment:
We learned to always return our fetch within an api function from a separate file.